dev/com.ibm.ws.session.cache/src/com/ibm/ws/session/store/cache/CacheHashMap.java

```diff
@@ -553,7 +664,15 @@ protected void insertSession(BackedSession session) {
      */
     @Override
     protected boolean isPresent(String id) {
-        return sessionInfoCache.containsKey(id);
+        final boolean trace = TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled();
```

Comment:
Just nitpicking, but it is kind of inconsistent that in some methods in this class trace = TraceComponent.isAnyTracingEnabled(); but in other methods trace = TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled();.  It may be easier to maintain in the future if you switch to one or the other, at least per class.

Reply:
updated